include/mbedtls/pk.h

```diff
@@ -201,7 +288,17 @@ void mbedtls_pk_free( mbedtls_pk_context *ctx );
  *                  MBEDTLS_ERR_PK_BAD_INPUT_DATA on invalid input,
  *                  MBEDTLS_ERR_PK_ALLOC_FAILED on allocation failure.
  *
- * \note            For contexts holding an RSA-alt key, use
+ * \note            Engines that implement opaque keys may offer an
+ *                  alternative setup function that take engine-dependent
+ *                  parameters. If such a function exists, call it
```

Comment:
Suggestion: “If such a function exists” → “If the documentation of the engine lists such a function”

Reply:
Done